docs/features/records.md

```diff
@@ -108,35 +174,44 @@ The programmer may provide an explicit constructor with the same signature as th
 At runtime the primary constructor
```

Comment:
They have a primary constructor corresponding to the one you declared. The modifiers on the record type no longer have any influence on the shape of the constructor.